### PR TITLE
[db-up] `az sql up`: Fix pymssql installation issue

### PR DESCRIPTION
**Pull request by evelyn-ys**
`az sql up` will fail when using Python 3.8 or Python 3.9 like this:
```
> az sql up -g yishitest -s ystestserver -d ystestdb -u ysadmin -p yspwd@123
Installing dependencies required to configure Azure SQL server...
  Installing "pymssql" pip packages
The command failed with an unexpected error. Here is the traceback:
Command '['D:\\workspace\\env39\\Scripts\\python.exe', '-m', 'pip', 'install', '--target', 'C:\\Users\\yishiwang\\.azure\\cliextensions\\db-up', 'pymssql==2.1.4', '-vv', '--disable-pip-version-check', '--no-cache-dir']' returned non-zero exit status 1.
Traceback (most recent call last):
  File "d:\workspace\cli-extension-for-codegen\src\db-up\azext_db_up\custom.py", line 197, in _ensure_pymssql
    import pymssql  # pylint: disable=unused-import,unused-variable
ModuleNotFoundError: No module named 'pymssql'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\workspace\env39\lib\site-packages\knack\cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "d:\workspace\azure-cli\src\azure-cli-core\azure\cli\core\commands\__init__.py", line 657, in execute
    raise ex
  File "d:\workspace\azure-cli\src\azure-cli-core\azure\cli\core\commands\__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "d:\workspace\azure-cli\src\azure-cli-core\azure\cli\core\commands\__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "d:\workspace\azure-cli\src\azure-cli-core\azure\cli\core\commands\__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "d:\workspace\azure-cli\src\azure-cli-core\azure\cli\core\commands\command_operation.py", line 121, in handler
    return op(**command_args)
  File "d:\workspace\cli-extension-for-codegen\src\db-up\azext_db_up\custom.py", line 145, in sql_up
    _ensure_pymssql()
  File "d:\workspace\cli-extension-for-codegen\src\db-up\azext_db_up\custom.py", line 215, in _ensure_pymssql
    subprocess.check_output(cmd, stderr=subprocess.STDOUT)
  File "C:\Python39\lib\subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "C:\Python39\lib\subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['D:\\workspace\\env39\\Scripts\\python.exe', '-m', 'pip', 'install', '--target', 'C:\\Users\\yishiwang\\.azure\\cliextensions\\db-up', 'pymssql==2.1.4', '-vv', '--disable-pip-version-check', '--no-cache-dir']' returned non-zero exit status 1.
```

There're 2 problems here:
- pymssql 2.1.4 is too old for Python 3.8/3.9
- Install dir is `.azure\azcliextensions\db-up` which only works when installing extension with `az extension add`, but can't work for installing extension from source code with `azdev extension add`

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
db-up

**Review by calvinhzy — APPROVED** [review]

File: src/db-up/setup.py [hunk 1]
```diff
@@ -28,6 +29,9 @@
     'psycopg2-binary==2.9.1'
 ]
 
+if sys.platform != 'darwin':
```
Comment (jiasli) [inline src/db-up/setup.py:32]: Why darwin doesn't need it?

File: src/db-up/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.2.6 (2021-09-08)
+++++++++++++++++++
+* `az sql up`: Fix pymssql installation issue
```
Comment (jiasli) [inline src/db-up/HISTORY.rst:7]: Please use more specific wording, instead of generic "issue".

Comment (evelyn-ys) [inline src/db-up/HISTORY.rst:7]:
```suggestion
* `az sql up`: Fix pymssql installation failure with Python 3.8/3.9
```

File: src/db-up/setup.py (hunk 1 above)
Comment (evelyn-ys) [reply src/db-up/setup.py:32]: Actually darwin also need. But the installation in darwin depends on `brew install freetds` first.

**Pull request merged**